self-hosted-runner.tf

```diff
@@ -0,0 +1,125 @@
+provider "aws" {
+  region = "eu-west-1"
+  access_key = var.aws_access_key
+  secret_key = var.aws_secret_key
+}
+
+variable "aws_access_key" {
+  description = "AWS Access Key"
+  type        = string
+}
+
+variable "aws_secret_key" {
+  description = "AWS Secret Key"
+  type        = string
+}
+
+variable "aws_key_name" {
+  description = "AWS Key Name"
+  type = string
+}
+
+resource "aws_instance" "self_hosted_runner" {
+  ami           = "ami-055032149717ffb30" # change to ami-0776c814353b4814d when creating an AMI. 
```

Comment:
What's the reason for changing the ami when creating a new one?

Reply:
The commented AMI ID is a ubuntu AMI. This image allows us to have a clean environment to set up our dependencies so we can use the newly created AMI later on to save time on setting up our testing environment.